servant-server/src/Servant/Server/Internal.hs

```diff
@@ -588,7 +608,7 @@ instance HasServer Raw context where
 -- The @Content-Type@ header is inspected, and the list provided is used to
 -- attempt deserialization. If the request does not have a @Content-Type@
 -- header, it is treated as @application/octet-stream@ (as specified in
--- <http://tools.ietf.org/html/rfc7231#section-3.1.1.5 RFC7231>.
+-- [RFC 7231 section 3.1.1.5](http://tools.ietf.org/html/rfc7231#section-3.1.1.5 RFC7231).
```

Comment:
Can you please keep the haddocks markup?  Or does haddock support markdown these days?

Reply:
![image](https://user-images.githubusercontent.com/275614/84600862-1e45dd80-ae85-11ea-9957-cb4811921c14.png)
It does support this particular syntax, yes.

There was however a problem in my variant, thanks for drawing my attention!

Reply:
See here: https://haskell-haddock.readthedocs.io/en/latest/markup.html#links